### ISSUE TITLE
API Gateway is not deployed on `zappa update`

### ISSUE THREAD

**Issue by claudeatsafe** [Miserlou/Zappa#382]
This means updates are applied but do not have any effect. It is mainly an issue if something changes related to Authorization but it also means the following parameters cannot be updated after the initial deploy:
```
cache_cluster_enabled
cache_cluster_size
cloudwatch_log_level
cloudwatch_data_trace
cloudwatch_metrics_enabled
```

**Comment by NathanLawrence** [comment]
Yep. There are a number of settings that remain unaffected by update for lambdas as well. Right now, the guidance is generally to redeploy. I know this is inconvenient, but luckily Zappa's domain handling makes this relatively smooth.

**Comment by claudeatsafe** [comment]
Since the CloudFormation stack is actually updated and Zappa supports this functionality, I guess the changes should also be applied. I'll open a PR.
